### PR TITLE
Change check NotInheritedAttribute

### PR DESCRIPTION
**Pull request by AlexandrTsvetkov**
Change check `InheritedAttribute` for classes `BaseB` and `DerivedB` to check `NotInheritedAttribute`

### CODE REVIEW

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you , @AlexandrTsvetkov, for contributing to the dotnet/docs repo and for correcting what I suspect was a copy-and-paste error in the code example.  There's also a Visual Basic example in dotnet/docs/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.AttributeUsageAttribute.Inherited/vb. Would you mind fixing that as well? Or if you prefer, we can make the changes for you.

File: samples/snippets/csharp/VS_Snippets_CLR_System/System.AttributeUsageAttribute.Inherited/cs/Inherited1.cs [hunk 1]
```diff
@@ -42,19 +42,19 @@ public static void Main()
                         memberA.GetCustomAttributes(typeof(InheritedAttribute), true).Length > 0); 
       
       Type typeB = typeof(DerivedB);
-      Console.WriteLine("DerivedB has Inherited attribute: {0}", 
-                        typeB.GetCustomAttributes(typeof(InheritedAttribute), true).Length > 0); 
+      Console.WriteLine("DerivedB has NotInherited attribute: {0}", 
+                        typeB.GetCustomAttributes(typeof(NotInheritedAttribute), true).Length > 0); 
       MethodInfo memberB = typeB.GetMethod("MethodB");
```
Comment (jzeferino) [inline samples/snippets/csharp/VS_Snippets_CLR_System/System.AttributeUsageAttribute.Inherited/cs/Inherited1.cs]: Would be nice to use `nameof` operator here: `MethodInfo memberB = typeB.GetMethod(nameof(DerivedB.MethodB));`

In line 40, I suggest the same change to `MethodInfo memberA = typeA.GetMethod("MethodA");`

**Review by jzeferino — CHANGES_REQUESTED** [review]

**Comment by AlexandrTsvetkov** [comment]
Thank you for your suggestion @rpetrusha . Unfortunately, I don't use Visual Basic and it will be much more better, if you fix it by yourself.

**Review by BillWagner — APPROVED** [review]
Thanks for fixing this @AlexandrTsvetkov We do appreciate the work.

@rpetrusha I made the VB updates, and also switched the C# code to use interpolated strings.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks again for opening this issue and correcting the C# code, @AlexandrTsvetkov, and thanks for correcting the VB code, @BillWagner. We'll merge your PR now. The changes should appear live on docs.microsoft.com in the next day or two.

**Pull request merged**